CMakeLists.txt

```diff
@@ -163,6 +163,8 @@ if(MSVC)
   add_definitions(-DDMLC_STRICT_CXX11)
   add_definitions(-DNOMINMAX)
   set(CMAKE_C_FLAGS "/MP")
+  # report an accurate value for recent C++ language standards support
+  set(CMAKE_CXX_FLAGS "/Zc:__cplusplus")
```

Comment:
@ChaiBapchya @vexilligera you added this line, but it doesn't have any effect as it is overwritte in the line below

Reply:
set(flag ${flag} "secondval") does append right?
I thought we are appending to CMAKE_CXX_FLAGS

Reply:
the typo!!!! `CMAKE_CXX_FLAGS` instead of `CMAKE_C_FLAGS`

Reply:
My bad. I thought it would append...